client/blocks/get-apps/style.scss

```diff
@@ -7,27 +17,140 @@
 	width: 100%;
 }
 
+.get-apps__wrapper.jetpack .card {
+	box-shadow: none;
+
+	@include breakpoint-deprecated("<480px") {
+		padding: 30px;
+	}
+
+}
+
 .get-apps__card-title {
 	font-size: $font-body;
 	margin-bottom: 4px;
 
+	&.jetpack {
+		font-size: $font-headline-small;
+		line-height: 41px;
+		@extend .wp-brand-font;
+	}
+
 	@include breakpoint-deprecated(">480px") {
 		font-size: $font-title-small;
+
+		&.jetpack {
+			font-size: $font-headline-medium;
+			line-height: 52px;
+		}
+	}
+}
+
+.get-apps__jetpack-branded-text {
+	color: var(--studio-jetpack-green-40);
+}
+
+.get-apps__jetpack-branded-link {
+	color: var(--studio-jetpack-green-40);
+	font-weight: 700;
+	cursor: pointer;
+
+	&:visited {
+		color: var(--studio-jetpack-green-40);
+	}
+
+	&:hover,
+	&:active,
+	&.is-active,
+	&:focus {
+		color: var(--studio-jetpack-green-60);
+		text-decoration: underline;
+		text-underline-offset: 3px;
 	}
 }
 
 .get-apps__desktop {
+
+	&.jetpack {
+		background: #e9eff5;
+		margin-bottom: 0;
+
+		&::before {
+			content: "";
+			background-color: #e9eff5;
+			position: absolute;
+			height: 100%;
+			width: 200vw;
+			left: -100vw;
+			z-index: -1;
+		}
```

Comment:
This is quite clever — I mean that in a positive way. That said, whenever we are able to remove the branding conditionals, it would likely be sensible to refactor the layout padding to make this absolutely positioned element unnecessary.

Reply:
Thank you, I agree it'd be nice to refactor so that everything is cleaner when we go about removing the branding conditionals.